data/country.csv

```diff
@@ -1,218 +1,226 @@
-Algeria,Algérie,,Algeria,الجزائر,2,2-PME,28.0000272,2.9999825,DZ,
-السعودية,Arabie Saoudite,,Saudi Arabia,السعودية,2,2-PME,25.6242618,42.3528328,SA,
-Argentina,Argentine,,Argentina,الأرجنتين,2,2-PME,-34.9964963,-64.9672817,AR,
-Australia,Australie,,Australia,أستراليا,2,2-PME,-24.7761086,134.755,AU,
-Brasil,Brésil,,Brazil,البرازيل,2,2-PME,-10.3333333,-53.2,BR,
-Cameroun - Cameroon,Cameroun,,Cameroon,الكاميرون,2,2-PME,4.6125522,13.1535811,CA,
-Canada,Canada,,Canada,كندا,2,2-PME,61.0666922,-107.9917071,CV,
-中国,Chine,,China,الصين,2,2-PME,35.000074,104.999927,CN,
-대한민국,Corée du Sud,,South Korea,كوريا الجنوبية,2,2-PME,36.5581914,127.9408564,KR,
-Côte d'Ivoire,Côte d'Ivoire,,Ivory Coast,كوت ديفوار,2,2-PME,7.9897371,-5.5679458,CI,
-مصر,Égypte,,Egypt,مصر,2,2-PME,26.2540493,29.2675469,EG,
-الإمارات العربية المتحدة,Émirats arabes unis,E.A.U,United Arab Emirates,الإمارات العربية المتحدة,2,2-PME,24.0002488,53.9994829,AE,
-ኢትዮጵያ / Ethiopia,Éthiopie,,Ethiopia,إثيوبيا,2,2-PME,10.2116702,38.6521203,ET,
-India,Inde,,India,الهند,2,2-PME,22.3511148,78.6677428,IN,
-Indonesia,Indonésie,,Indonesia,إندونيسيا,2,2-PME,-4.7993356,114.5632032,ID,
-ישראל,Israël,,Israel,فلسطين,2,2-PME,30.8760272,35.0015196,IL,
-日本,Japon,,Japan,اليابان,2,2-PME,36.5748441,139.2394179,JP,
-Kenya,Kenya,,Kenya,كينيا,2,2-PME,1.4419683,38.4313975,KE,
-لبنان,Liban,,Lebanon,الجمهورية اللبنانية,2,2-PME,33.8750629,35.843409,LB,
-México,Mexique,,Mexico,المكسيك,2,2-PME,22.5000485,-100.0000375,MX,
-Nigeria,Nigéria,,Nigeria,نيجيريا,2,2-PME,9.6000359,7.9999721,NG,
-‪‫پاکستان‬,Pakistan,,Pakistan,باكستان,2,2-PME,30.3308401,71.247499,PK,
-Polska,Pologne,,Poland,بولندا,2,2-PME,52.0977181,19.0258159,PL,
-Россия,Russie,,Russia,روسيا الاتحادية,2,2-PME,64.6863136,97.7453061,RU,
-ประเทศไทย,Thaïlande,,Thailand,تايلاند,2,2-PME,14.8971921,100.83273,TH,
-تونس,Tunisie,,Tunisia,تونس,2,2-PME,33.8439408,9.400138,TN,
-Türkiye,Turquie,,Turkey,تركيا,2,2-PME,38.9597594,34.9249653,TR,
-Україна,Ukraine,,Ukraine,أوكرانيا,2,2-PME,49.4871968,31.2718321,UA,
-France,France,,France,فرنسا,1,1-HC,46.603354,1.8883335,FR,
-Deutschland,Allemagne,,Germany,ألمانيا,1,1-PME/FE,51.0834196,10.4234469,DE,
-España,Espagne,,Spain,إسبانيا,1,1-PME/FE,40.0028028,-4.003104,ES,
-United States of America,États-Unis,,United States of America,الولايات المتّحدة الأمريكيّة,1,1-PME/FE,39.7837304,-100.4458825,US,
-Italia,Italie,,Italy,إيطاليا,1,1-PME/FE,42.6384261,12.674297,IT,
-Madagascar,Madagascar,,Madagascar,مدغشقر,1,1-PME/FE,-18.9249604,46.4416422,MG,
-Maroc ⴰⵎⵔⵔⵓⴽ المغرب,Maroc,,Morocco,المغرب,1,1-PME/FE,31.1728192,-7.3366043,MA,
-United Kingdom,Royaume-Uni,,United Kingdom,المملكة المتحدة,1,1-PME/FE,54.7023545,-3.2765753,GB,
-Sénégal,Sénégal,,Senegal,السنغال,1,1-PME/FE,14.4750607,-14.4529612,SN,
+name,name:fr,short_name:fr,name:en,name:ar,prio,type,latitude,longitude,iso,sov,wikidata,admin_level
+La Réunion,La Réunion,,Réunion,ريونيون,99,99-Other,-21.13093,55.52658,RE,FR,Q17070,4
```

Comment:
I feel like this is data pâté ;)
Country are by definition admin_level=2, mixin them with other territories would need at least a rename.
But the first of all why not using OSM data directly for those labels?

Reply:
The fact is that UK's overseas are admin_level=2 and not french ones... Even if french ones have their own ISO code also.

What I wanted is also to show out easlily from which sovereign country it depends (UK / FR / US / NO / DK).